lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -532,11 +532,14 @@ lldb::VariableSP SwiftExpressionParser::FindSelfVariable(Block *block) {
   return variable_list_sp->FindVariable(ConstString("self"));
 }
 
-static void AddRequiredAliases(Block *block, lldb::StackFrameSP &stack_frame_sp,
-                               SwiftASTContextForExpressions &swift_ast_context,
-                               SwiftASTManipulator &manipulator,
-                               lldb::DynamicValueType use_dynamic,
-                               lldb::BindGenericTypes bind_generic_types) {
+/// Adds the required type aliases to evaluate an expression. Returns a Status
+/// with an error message in case of failure.
```

Comment:
The second sentence is optional because it's more of a stretch goal.

```suggestion
/// - Returns: A `Status` instance that indicates whether the method finished successfully. If the method returns an error status, it contains a string that explain the failure.
```



The `///`-style documentation comments (aka doc-comments) have a dedicated `- Returns` section to explain the return value.